libs/ui/src/ui_strings/app_strings.tsx

```diff
@@ -402,6 +430,58 @@ export const appStrings = {
     <UiString uiStringKey="labelWriteInParenthesized">(write-in)</UiString>
   ),
 
+  letterA: () => <UiString uiStringKey="letterA">A</UiString>,
+
+  letterB: () => <UiString uiStringKey="letterB">B</UiString>,
+
+  letterC: () => <UiString uiStringKey="letterC">C</UiString>,
+
+  letterD: () => <UiString uiStringKey="letterD">D</UiString>,
+
+  letterE: () => <UiString uiStringKey="letterE">E</UiString>,
```

Comment:
Was curious how Google Translate was going to handle these single-letter values and it did generate wonky results for some 😆

<table>
<img width="400" alt="mistranslation-e" src="https://github.com/votingworks/vxsuite/assets/12616928/13110867-5031-467a-9a3b-63cd5079c462">
</table>

Not a big deal at all, just making me realize that even before we build proper override functionality, I'll want to bake some overrides in, maybe by preloading values into the VxDesign translation cache

Reply:
Haha! I'd initially forgotten to force English rendering and ended up with a very funky looking keyboard 😅 

And good point -- I keep forgetting about the override functionality and how that'll interact with some of these edge cases. That makes me think we might need two levels of overrides: one for changes VotingWorks makes when we audit the cloud translations and another on top of that the the election officials manage (in case we want them to be able to undo/remove their overrides for some strings). Or maybe Vx overrides/fixes are just applied directly to the base translations layer, since we can always just re-generate from GCloud if needed.

Reply:
Yeah good point that these two overrides are conceptually distinct!